### ISSUE TITLE
 Unknown compiler option 'lib'

### ISSUE THREAD

**Issue by buddyackerman** [Microsoft/TypeScript#11385]
I get this and the following error when trying to compile a new project that was created with angular-cli (using NG2 Final).

>Unknown compiler option 'typeRoots'.

  Here is the tsconfig:

> {
  "compilerOptions": {
    "declaration": false,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "lib": ["es6", "dom"],
    "mapRoot": "./",
    "module": "es6",
    "moduleResolution": "node",
    "outDir": "../dist/out-tsc",
    "sourceMap": true,
    "target": "es5",
    "typeRoots": [
      "../node_modules/@types"
    ]
  }
}

The typescript version in the project is 2.0.2.

**Comment by DanielRosenwasser** [comment]
Are you compiling with `tsc` from the command prompt? If you type in `tsc --version`, what do you get?

**Comment by buddyackerman** [comment]
@DanielRosenwasser Well, Dan, you pointed me to where the problem was.  I was in fact running tsc from the command line and of course that would run the globally installed version of tsc not the version installed locally in the project I created.  The global version was 1.8.10.  I updated my global installation to 2.0.3 and it no longer throws those errors (I get other errors but those aren't your problem).
